conformance/packages/dns-test/src/templates/hickory.resolver.toml.jinja

```diff
@@ -1,5 +1,4 @@
 [[zones]]
 zone = "."
 zone_type = "Hint"
-stores = { type = "recursor", roots = "/etc/root.hints", security_aware = true }
-enable_dnssec = {{ use_dnssec }}
```

Comment:
If my understanding is correct the `enable_dnssec` is not used by the `Resolver`, the DNSSEC logic is handledby the given `DnssecPolicy` here? Does this in any way affect the example`toml` files in `tests/test-data/test_configs` &  should they be updated as well?

Reply:
`enable_dnssec` is only used by the `file` and `sqlite` type stores (which must be used with the `Primary` and `Secondary` `zone_type`s). it is not used by the `recursor` (`Hint`) nor the `forward`-type stores. 

IOW, this `enable_dnssec` had no effect in this file. I added a while ago when I did not understand the structure of the `named.toml` file -- which then prompted me to suggest RFC #2195

So, you should not understand `enable_dnssec = true` must now be replaced with `dnssec_policy = Something` but rather as the `enable_dnssec`, which was a no-op, is getting removed (clean-up); *and* the use of `security_aware` is being replaced with `dnssec_policy`.

Therefore, the `toml` files under `test_configs` do not need to be updated as part of this PR.